vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -15,7 +15,15 @@ import { ItemCache } from '@vaadin/vaadin-grid/src/vaadin-grid-data-provider-mix
         return;
       }
 
+      // Storing original implementation of the method to be used for client
+      // side only grids
+      ItemCache.prototype.ensureSubCacheForScaledIndexOriginal = ItemCache.prototype.ensureSubCacheForScaledIndex;
```

Comment:
Wasn't sure how to call the stored method, suggestions are welcome.

Reply:
You mean if you stored it as `const ensureSubCacheForScaledIndexOriginal  = ...`?

Would it work if it was then called with `ensureSubCacheForScaledIndexOriginal.call(this, scaledIndex)`?

Reply:
My bad. I meant how to name the method. Not sure if "Original" at the end is good.

> Would it work if it was then called with ensureSubCacheForScaledIndexOriginal.call(this, scaledIndex)?

I thought of calling the method this way, but since the prototype patching is being done everywhere, I decided to go on the same path.